api/source/service/STIGService.js

```diff
@@ -148,7 +148,7 @@ exports.queryGroups = async function ( inProjection, inPredicates ) {
   
   if (inPredicates.revisionStr != 'latest') {
     joins = ['revision r']
-    let [results, version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(inPredicates.revisionStr)
+    let [version, release] = /V(\d+)R(\d+(\.\d+)?)/.exec(inPredicates.revisionStr)
```

Comment:
regex returns what it captures positionally, so we need "results" here, even though it is not used.